### PR TITLE
[sonic-yang-mgmt] Replace subprocess using with shell=True

### PR DESCRIPTION
**Pull request by maipbui**
Signed-off-by: maipbui <maibui@microsoft.com>

#### Why I did it
`subprocess` is used with `shell=True`, which is very dangerous for shell injection.
#### How I did it
remove `shell=True`, use `shell=False`
#### How to verify it
Pass UT
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205

### CODE REVIEW

File: src/sonic-yang-mgmt/tests/test_cfghelp.py [hunk 1]
```diff
@@ -126,8 +126,8 @@ def setUp(self):
         self.script_file = os.path.join(self.test_dir, '..', 'sonic-cfg-help')
 
     def run_script(self, argument):
-        print('\n    Running sonic-cfg-help ' + argument)
-        output = subprocess.check_output(self.script_file + ' ' + argument, shell=True)
+        print('\n    Running sonic-cfg-help ', argument)
```
Comment (qiluo-msft) [inline src/sonic-yang-mgmt/tests/test_cfghelp.py:129]: >, [](http://example.com/codeflow?start=45&length=1)

Semgrep alerts on this line?

Comment (maipbui) [reply src/sonic-yang-mgmt/tests/test_cfghelp.py:129]: No, but `argument` variable is a list, cannot be concatenated with string in the print statement.

Comment (qiluo-msft) [reply src/sonic-yang-mgmt/tests/test_cfghelp.py:129]:
```suggestion
        print('\n    Running sonic-cfg-help ' + shlex.join(argument))
```

Comment (maipbui) [reply src/sonic-yang-mgmt/tests/test_cfghelp.py]: Updated.

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**